### PR TITLE
[PyTorch] Add tests for quantized MaxPool and AvgPool

### PR DESCRIPTION
**Pull request by jackm321**
Summary:
No code changes since quantized pooling uses the same ops.

Documentation:
comments

Test Plan:
new tests
`python setup.py test`

### CODE REVIEW

File: torch_glow/tests/nodes/quantized_maxpool_test.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import torch
+
+from tests.utils import jitVsGlow
+
+
+def test_quantized_maxpool():
+    """Basic test of the PyTorch quantized::max_pool2d Node on Glow."""
+
+    def test_f(a):
+        q = torch.nn.quantized.Quantize(1/128, 3, torch.quint8)
```
Comment (rdzhabarov) [inline torch_glow/tests/nodes/quantized_maxpool_test.py]: 1/128 is 0, is that expected?

**Comment by zrphercule** [comment]
lol, even harder than I expected...

**Comment by zrphercule** [comment]
> LGTM. But param to the Quantize looks a bit fishy.

In fact 1/128 or 1/256 is what we supposed to give, since we are quantizating a 0~1 float to 0~255.

**Comment by rdzhabarov** [comment]
NVM my comment on 1/128.

1/128 seems behave differently in python 2 vs python 3.

python 2 => 1/128 == 0
python 3 does fp division.

**Comment by jackm321** [comment]
@rdzhabarov no it's a good catch, it should be explicit about it. I changed it. Thank you for catching it

**Comment by rdzhabarov** [comment]
> scale=1.0/128, zero_point=5, dtype=torch.quint8

Thanks, it's much more readable!

**Pull request closed without merging**